README.md

```diff
@@ -4,18 +4,19 @@ Extracts SourceMaps for source files that as added as `sourceMappingURL` comment
 
 ## Usage
 
-[Documentation: Using loaders](http://webpack.github.io/docs/using-loaders.html)
+[Documentation: Using loaders](https://webpack.js.org/concepts/#loaders)
 
 
-### example webpack config
+### Example webpack config
 
 ``` javascript
 module.exports = {
   module: {
-    preLoaders: [
+    rules: [
       {
         test: /\.js$/,
-        loader: "source-map-loader"
+        loader: "source-map-loader",
```

Comment:
For sure. I used [this](https://webpack.js.org/guides/migrating/#module-preloaders-and-module-postloaders-was-removed) as the example so you might want to update that as well.